### PR TITLE
Schema_Based_On_Diagram

### PR DESCRIPTION
**Pull request by Smart1-hub**
Create schema file based on diagram
Add fk index

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  treatment_id integer,
+  CONSTRAINT fk_invoice FOREIGN KEY(invoice_id) REFERENCES invoices(id),
+  CONSTRAINT fk_treatment FOREIGN KEY(treatment_id) REFERENCES treatments(id)
+);
+
+CREATE INDEX ON medical_histories (patient_id);
+CREATE INDEX ON invoices (medical_history_id);
+CREATE INDEX ON invoice_items (invoice_id);
+CREATE INDEX ON invoice_items (treatment_id);
+
```
Comment (sja-thedude) [inline schema_based_on_diagram.sql:58]: - [ ] Kindly make sure to add Index for medical_histories_has_treatments example:

```
CREATE INDEX ON medical_histories_has_treatments (medical_history_id);
CREATE INDEX ON medical_histories_has_treatments (treatment_id);
```

File: schema_based_on_diagram.sql [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  id INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+  unit_price decimal,
+  quantity integer,
+  total_price decimal,
+  invoice_id integer,
+  treatment_id integer,
+  CONSTRAINT fk_invoice FOREIGN KEY(invoice_id) REFERENCES invoices(id),
+  CONSTRAINT fk_treatment FOREIGN KEY(treatment_id) REFERENCES treatments(id)
+);
+
```
Comment (sja-thedude) [inline schema_based_on_diagram.sql:42]: - [ ] Kindly make sure that all "join table" for the many-to-many relationships is created, even if they are not showed on the diagram. Therefore students should create a table between `medical_histories` and `treatments`:
```sql
 CREATE TABLE medical_histories_has_treatments (
    medical_history_id int refrences medical_histories(id),
    treatment_id int refrences treatments(id),
    );
```

- [ ] Kindly make sure FOREIGN KEYS are set (by using `references table_name(id)` in column definition): like:
```
medical_history_id in medical_histories_has_treatments

treatment_id in medical_histories_has_treatments
```

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @Smart1-hub & @iBel251, This is @sja-thedude your code reviewer.

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

768070f3-bc9a-4aba-b8bc-25793e00ea1e_text

## Highlights
✔️ Professional Readme
✔️ Correct Gitflow

## Required 
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Smart1-hub — COMMENTED** [review]
Update reviewer's comments

**Review by Deepakdanger — APPROVED** [review]
Hi @Team, 

### Status: Approved ✔️

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚢 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**